src/g_editor.c

```diff
@@ -4341,7 +4341,7 @@ void canvas_connect(t_canvas *x, t_floatarg fwhoout, t_floatarg foutno,
 
         /* check if objects are already connected */
     if (canvas_isconnected(x, objsrc, outno, objsink, inno)) {
-        logpost(src, 3, "refusing duplicate connection");
+        logpost(src, 3, "this pair of iolets is already connected");
```

Comment:
Hah, now I feel like that's too much. Which pair are we referring to? It's not like it will select the two objects or the IOs directly to show the user. Also, would "iolet" be understandable to most people?

Maybe "inlet-outlet pair already connected" or "inlet/outlet already connect" or "inlet/outlet pair already connect"

Reply:
sure.
i'm offline now. please feel free to force-push.

Reply:
I made a small teak comment to the branch.